src/login.ts

```diff
@@ -641,7 +651,15 @@ export const login = {
         debug("Encoding SAML in base64");
         const samlBase64 = samlBuffer.toString("base64");
 
-        const url = `https://login.microsoftonline.com/${tenantId}/saml2?SAMLRequest=${encodeURIComponent(
+        console.log(`Azure AD authentication endpoint is: ${authenticationEndpoint}`);
+        // validate the azure ad authentication endpoint here as a common point whether
+        // configured in the profile, the profile was manually edited or we're falling
+        // back to the default. this is an additional security measure.
+        if (!AZURE_AD_AUTHENTICATION_ENDPOINTS.includes(authenticationEndpoint)) {
```

Comment:
This is a simple security preventative against a profile being improperly configured and pointed at an invalid or imposter endpoint. We could validate at the time of the `--configure` step but frankly something could just modify the profile after the fact.

Reply:
Let's also validate in the `--configure` step to avoid confusion

Reply:
I made the requested change. Let me know if my implementation is ok. Thanks.